src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue

```diff
@@ -56,6 +56,7 @@
 		</a>
 		<Actions
 			v-if="hasActions"
+			container="#content-vue"
```

Comment:
Is the navigation inside the content-vue?

Reply:
It is, but in fullscreen mode it's hidden by a property check. Probably by design.
I added this here just as a bonus.